.github/workflows/php-test-standalone-plugins.yml

```diff
@@ -0,0 +1,58 @@
+name: Standalone Plugin Integration Testing
+
+on:
```

Comment:
Does this workflow only work when the `plugins.json` file is updated? 

Reply:
@mukeshpanchal27 for now, it runs at the same time php unit tests run, but this will change once I have approval on the process itself. Thank you!